### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by idseefeld**
There should be a link for the new mode option.

### CODE REVIEW

File: Reference/Configuration/RuntimeSettings/index.md [hunk 1]
```diff
@@ -16,10 +16,15 @@ An example of a configuration could look something like:
...
+      "MaxRequestLength": 2048,
+      "Mode": "BackofficeDevelopment"
     }
   }
 }
 ```
 
 `MaxRequsetLength` is specified in kilobytes, so this configuration would limit requests, and therefore uploaded files, to 2 megabytes, and a maximum query string length of 90 characters.
+
+`Mode` could have three values: `BackofficeDevelopment` (default), `Development` and `Production`
```
Comment (eshanrnh) [inline Reference/Configuration/RuntimeSettings/index.md:28]:
```suggestion
`Mode` can have three values: `BackofficeDevelopment` (default), `Development`, and `Production`.
```

File: Reference/Configuration/RuntimeSettings/index.md [hunk 2]
```diff
@@ -16,10 +16,15 @@ An example of a configuration could look something like:
...
+
+Read more about the details at: [Fundamentals > Setup > Server-Setup > Runtime modes](/documentation/Fundamentals/Setup/Server-Setup/runtime-modes)
```
Comment (eshanrnh) [inline Reference/Configuration/RuntimeSettings/index.md:30]:
```suggestion
For more information, see the [Runtime modes](/documentation/Fundamentals/Setup/Server-Setup/runtime-modes) article.
```

**Review by eshanrnh — COMMENTED** [review]
Looks good, @idseefeld 🙌 Just 2 minor suggestions.

File: Reference/Configuration/RuntimeSettings/index.md (hunk 2 above)
Comment (idseefeld) [reply Reference/Configuration/RuntimeSettings/index.md:30]: 👍😁

Comment (eshanrnh) [reply Reference/Configuration/RuntimeSettings/index.md:30]: @idseefeld, 

If you agree with the suggestions, can you click on **Commit Suggestions** -> **Commit changes**. Once done, I'll merge in the PR 😃

**Comment by idseefeld** [comment]
Not sure I did the right way, but hopefully the last commit is ok now(?)

**Review by eshanrnh — APPROVED** [review]
Looks perfect, @idseefeld 🙌 Merging it in 💪

**Pull request merged**